housekeeping/docker/kagome-dev/minideb.Dockerfile

```diff
@@ -16,9 +16,6 @@ RUN apt-get update && \
         curl && \
         rm -rf /var/lib/apt/lists/*
 
-ENV LLVM_VERSION=11
```

Comment:
should this file be affected by revert?

Reply:
I accidentally committed some changes on it on the original commit, so the revert touches it, too.